package/regenda/package

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Copyright (c) 2023 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(regenda)
+pkgdesc="Caldav agenda app"
+url=https://github.com/tenJirka/reGenda
+pkgver=0.0.1-1
+timestamp=2023-08-23T19:40Z
+section="utils"
+maintainer="tenJirka <jirka.steh@gmail.com>"
+license=GPL3
+installdepends=(python3 python3-pip python3-lxml)
+flags=(patch_rm2fb)
+
+source=(https://github.com/tenJirka/reGenda/archive/refs/tags/0.0.1.zip)
+sha256sums=(00d6a1e2f6b59f86ba47ea5c3444f24e112d24eb62082802653fe4649153da01)
+
+package() {
+    install -D -m 644 -t "$pkgdir"/opt/etc/draft/ "$srcdir"/regenda.draft
+    install -D -m 644 -t "$pkgdir"/opt/etc/reGenda/ "$srcdir"/config.yml.example
+    install -D -m 644 "$srcdir"/icon.png "$pkgdir"/opt/etc/draft/icons/reGenda.png
+    install -D -m 755 -t "$pkgdir"/opt/reGenda/ "$srcdir"/run.py
+    install -D -m 644 -t "$pkgdir"/opt/reGenda/ "$srcdir"/languages.py
+    install -D -m 644 -t "$pkgdir"/opt/reGenda/ "$srcdir"/calendar_caldav.py
+    ln -s /opt/reGenda/run.py "$pkgdir"/opt/bin/reGenda
+}
+
+configure() {
+    pip install --upgrade pip caldav pyyaml pytz rm_pysas
```

Comment:
Especially since this upgrades pip, which is a system package.

`python3-pytz` is already a package that can be used. caldav, pyyaml, and rmpysas are not, and will need to be added.

Reply:
One option for you would be to vendor these files and store them alongside your scripts for now, instead of creating separate packages for all of them. You can do this as part of the build by `pip` installing them in the build process, and copying the python files/folders. That said, if they have any compiled portions, that will not work.

Reply:
I will look at that or I have an idea to install them via pip as written but not system wide but only to the directory to reGenda using `pip -t` ? Updating pip is not necessary as well. 
I just want to know if it can be an option.

Reply:
Should be ok now. I added all modules that cant be installed from toltec to package alongside script. I hope I managed to add all of them.